src/less/general.less

```diff
@@ -20,9 +20,20 @@ html.ext-wwt-popup body {
 
 .mw-parser-output .editor-token {
 	cursor: pointer;
+	position: relative;
+	display: inline-block;
+	white-space: pre;
 
-	&.active {
+	&.active::before {
+		content: '';
 		background-color: #facc48;
+		position: absolute;
+		top: 0;
+		left: 0;
+		z-index: -1;
+		width: 100%;
+		height: 100%;
+		padding-bottom: 6px;
```

Comment:
I'm getting a slightly odd display in Firefox, where the bottom padding goes downwards too far, e.g.:
![highlight-overlap](https://user-images.githubusercontent.com/213655/65662923-0cf34180-e068-11e9-899b-5b66ebb70d58.png)

This goes away with `top: -3px; padding-bottom: 3px` but I'm not sure what else that breaks.